dmt/main/models.py

```diff
@@ -1406,7 +1406,7 @@ def add_reply(self, user, body):
         self.email_reply(body, user, n)
 
     def email_reply(self, body, user, reply):
-        if self.author == user:
+        if self.user.userprofile == user:
```

Comment:
minor note - the "user" variable is a bit overloaded here. changing the function parameter name from "user" to "userprofile" would clarify the interaction here.